src/packages/cli/src/args.ts

```diff
@@ -1,16 +1,18 @@
 import { TruffleColors } from "@ganache/colors";
+
```

Comment:
I added a section in the README (directly _after_ the startup options) because I didn't feel it fit especially well with the existing options, mostly because of this line:

```The following options are listed for command line use, but can also be used when running Ganache programatically in your project.```

Please let me know what you think of how I've done this, and my wording.